Sources/SwiftFormat/Rules/OrderedImports.swift

```diff
@@ -47,6 +56,10 @@ public final class OrderedImports: SyntaxFormatRule {
       // Perform linting on the grouping of the imports.
       checkGrouping(linesSection)
 
+      if let firstLine = fileHeader.first, firstLine.type == .blankLine {
+        fileHeader.removeFirst()
+      }
```

Comment:
This handles the case in `testPreservesHeaderCommentInConditionalCompilationBlock`.

Previously, the newline separating the comment and `#if FOO` was printed as is as part of `fileHeader` resulting in an extra newline separating the comment and `#if FOO`.

Reply:
@allevato Actually looking at this again, the pretty-printer will always remove that line anyway (and looks like that isn't configurable), so there should be no breaking changes here.